doc/reference/reference_lua/box_info/synchro.rst

```diff
@@ -0,0 +1,142 @@
+            With elections enabled, an instance runs ``box.ctl.promote()`` command automatically after winning the elections.
+
+        -   ``term`` (since version :doc:`2.10.0 </release/2.10.0>`) -- current queue term.
+            It contains the term of the last ``PROMOTE`` request.
+            Usually, it is equal to :ref:`box.info.election.term <box_info_election>`.
+            However, the queue term value may be less than the corresponding one in the election term.
+            It can happen when a new round of elections started, but no instance has been promoted yet.
+
+        -   ``len`` -- current number of entries that are waiting in the queue.
+
+        -   ``busy`` (since version :doc:`2.10.0 </release/2.10.0>`) -- the boolean value is ``true``
+            when the instance is processing or writing some system request that modifies the queue
+            (for example, ``PROMOTE``, ``CONFIRM``, or ``ROLLBACK``).
+            Until the request is complete, any other incoming synchronous transactions and system requests
+            will be delayed.
+
+    *   ``quorum`` -- evaluated value of the
+        :ref:`replication_synchro_quorum <cfg_replication-replication_synchro_quorum>` configuration option.
+        Since version :doc:`2.5.3 </release/2.5.3>`, the option can be set as a dynamic formula.
+        In this case, the value of the ``quorum`` member depends on the current number of replicas.
+
+    **Example 1:**
+
+    In this example, the ``quorum`` field is equal to ``1``.
+    That is, synchronous transactions work like asynchronous ones.
+    `1` means that a successful WAL writing to the master is enough to commit.
+
+    ..  code-block:: tarantoolsession
+
+        tarantool> box.info.synchro
+        ---
+        - queue:
+            owner: 1
+            term: 2
+            len: 0
+            busy: false
+          quorum: 1
+        ...
+
+    **Example 2:**
+
+    First, set a quorum number and a timeout for synchronous replication using the following command:
+
+    ..  code-block:: tarantoolsession
+
+        tarantool> box.cfg{
+                 > replication_synchro_quorum=2,
+                 > replication_synchro_timeout=1000
+                 > }
+
+    Next, check the current state of synchronous replication:
+
+    ..  code-block:: tarantoolsession
+
+        tarantool> box.info.synchro
+        ---
+        - queue:
+            owner: 1
+            term: 2
+            len: 0
+            busy: false
+          quorum: 2
+        ...
+
+    Create a space called ``sync`` and enable synchronous replication on this space.
+    Then, create an index.
+
+    ..  code-block:: tarantoolsession
+
+        tarantool> s = box.schema.space.create("sync", {is_sync=true})
+        tarantool> _ = s:create_index('pk')
+
+    After that, use ``box.ctl.promote()`` function to claim a queue:
+
+    ..  code-block:: tarantoolsession
+
+        tarantool> box.ctl.promote()
+
+    Next, use the ``replace`` command:
```

Comment:
"Use the replace command" is correct but doesn't reflect a user's action. Let's rewrite this to smth like "Insert some tuples/data ...", "perform data maniupulations" or whatever you like :)